llvm/test/MC/AMDGPU/hsa-cov-invalid.s

```diff
@@ -0,0 +1,6 @@
+// RUN: not llvm-mc -triple amdgcn-amd-amdpal -mcpu=gfx802 %s 2>&1 | FileCheck %s
```

Comment:
Many times it's clearer if the negative test is grouped with positive tests. Grep `defsym ERR` for some examples.